learn/user-guide/testing-ballerina-code/mocking.md

```diff
@@ -84,50 +84,45 @@ import ballerina/io;
 import ballerina/http;
 import ballerina/regex;
 
-http:Client clientEndpoint = check new("https://api.chucknorris.io/jokes/");
+http:Client clientEndpoint = check new ("https://api.chucknorris.io/jokes/");
 
 // This function performs a `get` request to the Chuck Norris API and returns a random joke 
 // or an error if the API invocations fail.
 function getRandomJoke(string name, string category = "food") returns @tainted string|error {
     string replacedText = "";
-    var response = clientEndpoint->get("/categories");
+    http:Response response = check clientEndpoint->get("/categories");
 
     // Check if the provided category is available
-    if (response is http:Response) {
-        if (response.statusCode == http:STATUS_OK) {
-            var categories = response.getJsonPayload();
-
-            if (categories is json[]) {
-                if (!isCategoryAvailable(categories, category)) {
-                    error err = error("'" + category + "' is not a valid category.");
-                    io:println(err.message());
-                    return err;
-                }
+
+    if (response.statusCode == http:STATUS_OK) {
+        json categories = check response.getJsonPayload();
+
+        if (categories is json[]) {
```

Comment:
Can't we get rid of this check by assigning categories to a json[] ?

Reply:
Yes. I changed it to cast it to json[] array directly